Example/Tests/TestHelper.swift

```diff
@@ -30,3 +30,31 @@ class TestHelper: NSObject {
         TestHelper.shared.mockTracker.addMock(name, returnValue: returnValue)
     }
 }
+
+extension Dictionary where Key: Comparable {
```

Comment:
Do you mean to extend only where the Key is Comparable?

```suggestion
extension Dictionary where Key == Comparable {
```

Reply:
Does not work anyway 🤦